### ISSUE TITLE
Unidentified error when adding references to reply.tex

### ISSUE THREAD

**Issue by jeveki** [reproducible-reporting/templates#13]
Whenever I try to add a bibliography to reply.tex using `\bilbiography{references}`, I get following error requesting to open an issue. Log file is attached. The references.bib file is present in the same folder as reply.tex. It is not clear to me how to solve this.
[reply.log](https://github.com/reproducible-reporting/templates/files/13558616/reply.log)

**Comment by tovrstra** [comment]
The instructions printed on screen are not so helpful. The file with the error is reply.blg, which is a log file but without the `.log` extension. Can you upload that one or paste the text in a comment?

**Comment by tovrstra** [comment]
Part of the problem is addressed with the following change: https://github.com/reproducible-reporting/reprepbuild/commit/3eb48ca3b08b6946246bebdda33dc2dff8254427

**Comment by jeveki** [comment]
So the `reply.blg` file mentions that the `bibstyle` command is missing. I can resolve the issue by copying the following code from the `achemso.cls` to `ugent-note.cls`.

```
\newif\ifacs@biochem

\newcommand*\acs@bibstyle{achemso}
\ifacs@biochem
  \def\acs@bibstyle{biochem}
  \bibliographystyle{biochem}
\else
  \bibliographystyle{achemso}
\fi
\newcommand*\acs@bibliographystyle{}
\let\acs@bibliographystyle\bibliographystyle
\def\bibliographystyle#1{%
  \acs@warning{\string\bibliographystyle\space ignored}%
  \expandafter\acs@bibliographystyle\expandafter{\acs@bibstyle}%
}
````

However, the citations are now in an undesired format, which is probably related to this `biochem` key. I tried playing around with that key based on the `achemso.cls` file, but I found no immediate solution yet.

**Comment by jeveki** [comment]
I indeed solved it by adding ` \bibliographystyle{unsrt}` to `ugent-note.cls`.

**Comment by tovrstra** [comment]
The missing `\bibstyle` is now correctly recognized by RepRepBuild, fixed in reproducible-reporting/reprepbuild@a4bf8553f306f1c7184b1550aefbbd09d284b918. New release is on its way...
